src/user-guide/installing-workflows.rst

```diff
@@ -0,0 +1,370 @@
+.. _Installing-workflows:
+
+Installing Workflows
+====================
+
+Cylc commands identify workflows via their names, which are relative path names
+under the :term:`cylc-run directory`, ``~/cylc-run/`` by default.
+
+Workflows can be grouped together under sub-directories. E.g.:
+
+.. code-block:: console
+
+   $ cylc scan --state=all --name nwp --format=tree
+   nwp
+   ├─oper
+   │ ├─ region1
+   │ └─ region2
+   └─test
+      └─ region1
+
+This chapter will demonstrate how to install a workflow from an arbitrary
+location, called a :term:`source directory`.
+``cylc install`` will create a new directory in the :term:`cylc-run directory`
+for each installation of a workflow.
+
+.. _Install-Workflow:
+
+The Cylc Install Command
+------------------------
+
+Workflow names can be installed with the ``cylc install`` command,
+which creates the :term:`run directory` structure and some service
+files underneath it. Otherwise, ``cylc play`` will do this at workflow start
```

Comment:
Does Cylc Play do this?

Reply:
I will rephrase this, I meant cylc play creates the service files.